plottable.d.ts

```diff
@@ -2160,6 +2160,17 @@ declare module Plottable {
             constructor(colorScale: Scales.Color);
             protected _setup(): void;
             /**
+             * Gets the Formatter for the text.
+             */
+            formatter(): Formatter;
+            /**
+             * Sets the Formatter for the text.
```

Comment:
Hmm maybe be more specific to which text you're referring?  Maybe
"Sets the Formatter for the entries (or entry texts)."?  I would ping to get a second opinion